### PR TITLE
Timeout for subscription tests

### LINKED ISSUE #21: Add timeout for subscription tests
If the tests fail to publish a message for some reason, the subscriber tests will wait indefinitely for a message. Would be nice if the subscription tests had a timeout so a faulty test doesn't stop the execution of all other tests.

Perhaps use an rspec around hook: https://www.relishapp.com/rspec/rspec-core/v/3-3/docs/hooks/around-hooks

### PR DESCRIPTION
**Pull request by roback**
fix #21

### CODE REVIEW

File: spec/twingly/subscription_spec.rb [hunk 1]
```diff
@@ -1,6 +1,15 @@
 require "amqp_queue_context"
+require "timeout"
 
 describe Twingly::AMQP::Subscription do
+  SUBSCRIPTION_TIMEOUT = 5
+
+  around(:example) do |example|
```
Comment (walro) [inline spec/twingly/subscription_spec.rb]: `:example` -> `:each` ?

Comment (roback) [inline spec/twingly/subscription_spec.rb]: They used `:example` in the [documentation](https://www.relishapp.com/rspec/rspec-core/v/3-3/docs/hooks/around-hooks).

Apparently `:each` is just an alias for `:example` according to https://www.relishapp.com/rspec/rspec-core/v/3-3/docs/hooks/before-and-after-hooks.

Comment (roback) [inline spec/twingly/subscription_spec.rb]: I'll switch since we use `:each` in other places.

**Comment by jage** [comment]
:+1:

**Pull request merged**